ethsigner/signer/azure/src/test/java/tech/pegasys/ethsigner/signer/azure/AzureKeyVaultAuthenticatorTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.azure;
+
+public class AzureKeyVaultAuthenticatorTest {
+  /*
```

Comment:
Why is this commented out?

Reply:
Because it requires connection to the cloud vault - which is not going to be possible for testing.